caffe2/contrib/aten/gen_op.py

```diff
@@ -77,7 +77,7 @@ def value_is_tensor_type(v):
     'at::Scalar': 'assignTo(Output(${offset}),${output}.type(), ${output});',
     'bool': 'assignToValue<int64_t>(Output(${offset}),${output});',
     'int64_t': 'assignToValue<int64_t>(Output(${offset}),${output});',
-    'std::vector<at::Tensor>': 'assignListStartingAt(${offset}, ${output});',
+    '::std::vector<at::Tensor>': 'assignListStartingAt(${offset}, ${output});',
```

Comment:
this was terrible; we have a native operator called `std`, so the compiler confused this with `at::_ops::std` 😛

Reply:
This is indeed terrible lol. Wonder if we should munge the `at::_ops::` names haha